0002-data-collection-extension/README.md

```diff
@@ -1,96 +1,129 @@
 ---
-name: Data Collection Extension protocol
+name: Data Collection extension protocol
```

Comment:
This way it looks as if we're talking about an extension of Data Collection protocol. But the first line says DCX is an extension of Kaa protocol. I think we should shorten the title to `Data Collection extension` and keep it as per the adopted acronym to avoid redundancy and possible confusion.
Also, I don't see why we should capitalize common words like protocol, service, extension. Especially since our documentation style is towards the least possible amount of capitalization.